<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="socketListen">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>侦听</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IPComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>59</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本机 IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverStateLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="localPortEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>23333</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="sessionList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>460</width>
      <height>160</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>客户端</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>连接IP</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>连接端口号</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
